executor/executor_test.go

```diff
@@ -1858,6 +1858,26 @@ func (s *testSuite) TestMiscellaneousBuiltin(c *C) {
 	_, err = tidb.GetRows(rs)
 	c.Assert(err, NotNil)
 }
+
```

Comment:
Thx, so 'place in executor_test.go' in https://pingcap.com/blog-reconstruct-built-in-function-report-zh is out of date?